### PR TITLE
CustomMetadataIssueFixes

### PR DESCRIPTION
**Pull request by ShubhamVijaivargiya**
## Summary
Issue Fixes:
1. Changed fieldManageability for the following custom fields on the custom metadata-
a. Endpoint_Api_Version__c
b. Endpoint_Path__c
c. Merchant_Account__c
d. Single_Currency_Code__c
e. System_Integrator_Name__c

2. Introduced default values for the following fields:
a. Endpoint_Api_Version__c
b. Endpoint_Path__c

Enhancements:
1. Added page layout for the custom metadata Adyen Adapter
 
## Tested scenarios

**Fixed issue**:

### CODE REVIEW

**Review by maassenbas — APPROVED** [review]

**Review by vkuncham — APPROVED** [review]

File: adyen-appexchange/main/default/layouts/Adyen_Adapter__mdt-Adyen Adapter Layout.layout-meta.xml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+                <behavior>Edit</behavior>
+                <field>Single_Currency_Code__c</field>
+            </layoutItems>
+            <layoutItems>
+                <behavior>Edit</behavior>
+                <field>System_Integrator_Name__c</field>
+            </layoutItems>
+            <layoutItems>
+                <behavior>Edit</behavior>
+                <field>Service_Username__c</field>
```
Comment (vkuncham) [inline adyen-appexchange/main/default/layouts/Adyen_Adapter__mdt-Adyen Adapter Layout.layout-meta.xml:65]: Please remove this field, this field does not exist on custom metadata.

File: adyen-appexchange/main/default/layouts/Adyen_Adapter__mdt-Adyen Adapter Layout.layout-meta.xml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+            </layoutItems>
+            <layoutItems>
+                <behavior>Edit</behavior>
+                <field>Service_Password__c</field>
```
Comment (vkuncham) [inline adyen-appexchange/main/default/layouts/Adyen_Adapter__mdt-Adyen Adapter Layout.layout-meta.xml:69]: Please remove this field, this field does not exist on custom metadata.

File: adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Path__c.field-meta.xml [hunk 3]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>Endpoint_Path__c</fullName>
+    <defaultValue>&quot;/checkout&quot;</defaultValue>
```
Comment (vkuncham) [inline adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Path__c.field-meta.xml:4]: This change is not needed as we already populate this value here: https://github.com/Adyen/adyen-salesforce-b2b-lightning/blob/PW-6574-B2BLPackaging/adyen-appexchange/main/default/customMetadata/Adyen_Adapter.AdyenDefault.md-meta.xml#L23. I created the package without this change.

File: adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Api_Version__c.field-meta.xml [hunk 4]
```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
     <fullName>Endpoint_Api_Version__c</fullName>
+    <defaultValue>&quot;/v68&quot;</defaultValue>
```
Comment (vkuncham) [inline adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Api_Version__c.field-meta.xml:4]: This change is not needed as we already populate this value here: https://github.com/Adyen/adyen-salesforce-b2b-lightning/blob/PW-6574-B2BLPackaging/adyen-appexchange/main/default/customMetadata/Adyen_Adapter.AdyenDefault.md-meta.xml#L15. I created the package without this change.

File: adyen-appexchange/main/default/layouts/Adyen_Adapter__mdt-Adyen Adapter Layout.layout-meta.xml (hunk 1 above)
Comment (ShubhamVijaivargiya) [reply adyen-appexchange/main/default/layouts/Adyen_Adapter__mdt-Adyen Adapter Layout.layout-meta.xml]: Done

File: adyen-appexchange/main/default/layouts/Adyen_Adapter__mdt-Adyen Adapter Layout.layout-meta.xml (hunk 2 above)
Comment (ShubhamVijaivargiya) [reply adyen-appexchange/main/default/layouts/Adyen_Adapter__mdt-Adyen Adapter Layout.layout-meta.xml]: Done

File: adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Path__c.field-meta.xml (hunk 3 above)
Comment (ShubhamVijaivargiya) [reply adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Path__c.field-meta.xml:4]: This is a harmless change and also this exists for other fields(https://github.com/Adyen/adyen-salesforce-b2b-lightning/blob/PW-6574-B2BLPackaging/adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Authorize_Endpoint__c.field-meta.xml), so lets keep this for consistency.

File: adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Api_Version__c.field-meta.xml (hunk 4 above)
Comment (ShubhamVijaivargiya) [reply adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Api_Version__c.field-meta.xml:4]: This is a harmless change and also this exists for other fields(https://github.com/Adyen/adyen-salesforce-b2b-lightning/blob/PW-6574-B2BLPackaging/adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Authorize_Endpoint__c.field-meta.xml), so lets keep this for consistency.

Comment (vkuncham) [reply adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Api_Version__c.field-meta.xml:4]: Can you please remove this code ? As discussed over MM, the packaging has been done without this code and hence to keep things consistent, please remove this code from the repo.

File: adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Path__c.field-meta.xml (hunk 3 above)
Comment (vkuncham) [reply adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Path__c.field-meta.xml:4]: Can you please remove this code ? As discussed over MM, the packaging has been done without this code and hence to keep things consistent, please remove this code from the repo.

File: adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Api_Version__c.field-meta.xml (hunk 4 above)
Comment (ShubhamVijaivargiya) [reply adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Api_Version__c.field-meta.xml]: Done

File: adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Path__c.field-meta.xml (hunk 3 above)
Comment (ShubhamVijaivargiya) [reply adyen-appexchange/main/default/objects/Adyen_Adapter__mdt/fields/Endpoint_Path__c.field-meta.xml]: Done

**Pull request merged**